Update grammar, links, deprecations, etc in various text files (#2000)
GeyserMC/Geyser: README.md

## What is Geyser?
Geyser is a proxy, bridging the gap between Minecraft: Bedrock Edition and Minecraft: Java Edition servers.

> This project's ultimate goal is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**

Reviewer: Seems silly.

Reviewer 2: I wonder if it seems silly, im not sure perhaps I need another confirmation

Reviewer 3: It's worse haha

Author: How is it silly/worse lol. There's no difference and gets to the point quicker

Reviewer 4: yeah original sounds better

Author: ```suggestion
The ultimate goal of this project is to allow Minecraft: Bedrock Edition users to join Minecraft: Java Edition servers as seamlessly as possible. **Please note, this project is still a work in progress and should not be used on production. Expect bugs!**
```

- Structure block UI

## What can't be fixed

> We can't fix the following things because of Bedrock's limitations. They might be fixable in the future, but not as of now.

Reviewer 5: I would not write "we" because these things cannot be changed by any translation software

Reviewer: +1

Author: Yes.. that's covered by the "because of Bedrock's limitations"

Reviewer: The current wording doesn't place the blame on us exclusively. The listed things cannot be fixed by anyone, nor by anyone except the Bedrock developers.

Author: "The following things can only be fixed by Bedrock developers. They might be fixable in the future, but not as of now."

Reviewer: Let's not blame the Bedrock developers. 

"The following things cannot be fixed without changes to Bedrock. As of now, they are not fixable in Geyser." or something.

Author: ```suggestion
The following things cannot be fixed without changes to Bedrock. As of now, they are not fixable in Geyser.
```

## Compiling
1. Clone the repo to your computer
2. [Install Maven](https://maven.apache.org/install.html)
3. Navigate to the Geyser root directory and run `git submodule update --init --recursive`. This command downloads all the needed submodules for Geyser and is a crucial step in this process.
4. Run `mvn clean install` and locate to the `target` folder.

> 

Reviewer 6: Maybe you should add that if your having issues compiling try adding -U into the build command
E.g mvn clean install -U

Also you could add that adding -T 2C where 2 is the number of threads or cores (idk which one) maven uses

Author: @Camotoy thoughts? Barely out of the scope of this PR

Reviewer: I would agree that is out of the scope of this PR. It's not a bad idea, however.
